packages/svelte2tsx/src/htmlxtojsx_v2/utils/node-utils.ts

```diff
@@ -174,3 +174,43 @@ export function isShortHandAttribute(attr: Node): boolean {
 export function isQuote(str: string): boolean {
     return str === '"' || str === "'";
 }
+
+/*
+ * Found if there's a member access trailing behind an expression
+ * Usually it's there because of the preprocess we have before svelte parses
+ * the template
+ */
+export function getNodeEndIncludingTrailingPropertyAccess(
```

Comment:
Should I import from the v1 utils? I found there's some utility copied from v1. Maybe the plan is to remove v1 eventually this way it's easier to remove?

Reply:
Yeah the plan is to remove v1 at some point, so it's okay to duplicate it here